### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ngodi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+             <p>Gas That Makes a Mountain Breathe Fire Is Turning Up Around the World</p>
+             <p>Trump Objects to Mueller Testifying Before Congress</p>
+             <p>Menachem Mendel Taub, Grand Rabbi and Holocaust Survivor, Dies at 96</p>
+          </div>
+        </div>
+        <div class="advert advert_6">
+           <h2>ADVERT</h2>
+          </div>
+
+       <div class="details details_4 details_6">
```
Comment (AdnanAfsari) [inline index.html]: use footer tag instead of div in here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+          <div class="div_title">
+            <h3>The New York Times</h3>
+            <p class="figcap" >Go to Home Page >></p>
+          </div>
+          <div class="div_content">
+            <div class="content_items">
+              <h5>NEWS</h5>
+              <p>Home Page</p>
```
Comment (AdnanAfsari) [inline index.html]: if you check the original website you will find that this items are links so put them into a tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+                        <p>Sports</p>
+                        <p>Obituaries</p>
+                        <p>Today's Paper</p>
+                        <p>Corrections</p>
+                      </div>
+          </div>
+
+          <div class="div_footer">
+             <ul>
+               <li>© 2019 The New York Times Company</li>
```
Comment (AdnanAfsari) [inline index.html]: same put them into a tag.

**Comment by AdnanAfsari** [comment]
Hi @ngodi :(
The Theory Of Inflation image is placed perfectly now but you increased the width of the body which was ok before.

Can you please look at the original page and make exactly like that one.
if you can decrease the width of the body like before but keep the Theory Of Inflation image in the right place, the project will be done.

submit for another review after fixing the issue.please reach out to me if you need any help.

Happy Coding :)

**Comment by AdnanAfsari** [comment]
Hello @ngodi , Great work man 👍 
I really appreciate your hard work, you can merge the branch now.

Happy Coding :)

**Comment by ngodi** [comment]
Thank you @Adnan

**Pull request merged**